seed/pmintegration/manager.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python
```

Comment:
Not sure exactly where this file should live; probably not needed in a separate folder.  I guess really this would become a model eventually?  So drop it in seed/models?